### ISSUE TITLE
Allow user to specify `Validation Set` when starting

### ISSUE THREAD

**Issue by PaulCCCCCCH** [HaohanWang/Robustar_implementation#99]
Currently, we only allow users to give `Train Set` and `Test Set` when starting Robustar. We also want the user to mount a validation set into dockers `Robustar2/dataset/validation`. This argument should also be *required*. In its tooltip/help string/documentation, we can tell the users to use the same path for `Test Set` and `Validation Set` if they don't need a validation set.

We need to change the following:
1. The launcher.
2. `scripts/robustar.sh`.
3. `scripts/example.sh`.

To test it locally, when we start `Robustar` with a dummy validation set, make sure we can see those images in `ImageList`.
